src/packages/@ncigdc/components/LoadableWithLoading/Loading.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Overlay from '@ncigdc/uikit/Overlay';
+import Spinner from '@ncigdc/uikit/Loaders/Material';
+
+const Loading = ({ isLoading, timedOut, pastDelay, error }) => {
```

Comment:
timedOut will never be true, you need to pass a value for the timeout to the Loadable object in index.js below.
[npmdocs]( https://www.npmjs.com/package/react-loadable#timing-out-when-the-loader-is-taking-too-long)  [githubdocs4.0.2version](https://github.com/thejameskyle/react-loadable/tree/v4.0.2#optstimeout)

Reply:
didn't set a timeout because I think it's better for a page to eventually load then timeout when it would have eventually resolved...at least for this project, we end up with tickets about timeouts. :woman_shrugging: But we can definitely set one if necessary.